### PR TITLE
Add delete job button to jobs page.

### PR DESCRIPTION
**Pull request by inferno-chromium**
This deletes the job and removes associations in Fuzzer and
FuzzerJob entities.

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Didn't we not have this in the past because deleting a job can mess many things up?

**Comment by inferno-chromium** [comment]
No, we didn't do the cleanup properly, now it does by removing fuzzer and fuzzerjob mappings. i skipped template deletion, because that is even more confusing as it can uses by jobs and i have no idea on how it impacts. deleting a job on the other hand should be ok with this cleanup.

File: src/appengine/handlers/jobs.py [hunk 1]
```diff
@@ -41,7 +43,7 @@ def get_queues():
 
 
 class Handler(base_handler.Handler):
-  """Manage sets of environment variables for bots."""
+  """Return list of jobs and templates."""
```
Comment (jonathanmetzman) [inline src/appengine/handlers/jobs.py]: The new language seems a bit like it is for a comment instead of a class.

Comment (inferno-chromium) [reply src/appengine/handlers/jobs.py]: Done

**Pull request merged**